beta/src/pages/learn/sharing-state-between-components.md

```diff
@@ -137,10 +137,10 @@ export default function Accordion() {
     <>
       <h2>Almaty, Kazakhstan</h2>
       <Panel title="About" isActive={true}>
-        With a population of about 2 million, Almaty is Kazakhstan's largest city. From 1929 to 1997, it was its capital city.
+        २० लाख की आबादी के साथ, अलमाती कज़ख़िस्तान का सबसे बड़ा शहर है। १९२९ से १९९७ तक, वह उसकी राजधानी थी।
       </Panel>
       <Panel title="Etymology" isActive={true}>
-        The name comes from <span lang="kk-KZ">алма</span>, the Kazakh word for "apple" and is often translated as "full of apples". In fact, the region surrounding Almaty is thought to be the ancestral home of the apple, and the wild <i lang="la">Malus sieversii</i> is considered a likely candidate for the ancestor of the modern domestic apple.
+        यह नाम <span lang="kk-KZ">алма</span> से आता है, जो "सेब" का कज़ाख शब्द है और अक्सर "सेब से भरपूर" में अनुवाद करता है। असल में, अलमाती के आस पास का क्षेत्र को सेब का पैतृक घर माना जाता है, और जंगली <i lang="la">मालुस सिवेर्सि</i> को सेब के पूर्वज होने का संभावित उम्मीदवार माना जाता है।
```

Comment:
`मालुस सिवेर्सि` => `Malus sieversii`

lets keep as is. its a bit difficult to read in hindi